### ISSUE TITLE
I cannot use this version with webpack on MacOS

### ISSUE THREAD

**Issue by seeLuck** [madrobby/zepto#1262]
The module cannot be plugged in by webpack, so that the zepto is not defined.

but the v1.1.7 is ok with that. 

I can hardly understand the source code but it seems to be some modules issues with v1.2.0.

**Comment by mislav** [comment]
There was a patch proposing to fix this: https://github.com/madrobby/zepto/pull/1211

Can you test if that works?

**Comment by ohcibi** [comment]
I can confirm that this fix works. I.e. I built a project with a script that inserts an html element with the zepto. Without this fix I'm getting the expected error caused by `this` not being `window`.

@mislav I have an additional question about this. I found various (very old) issues about the zepto-team not wanting to support any kind of module loaders or similar like webpack (those issues were about browserify); but your answer on this issue occurs to me as if that has changed? I found the current version of zepto to be incompatible with browserify as well and I was wondering if issues about those topics are accepted nowadays?
